modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/src/main/resources/META-INF/resources/js/edit_blueprint/PreviewSidebar.js

```diff
@@ -53,6 +53,30 @@ function PreviewSidebar({
 		_handleFetch();
 	}, [activeDelta, activePage]);
 
+	const _copyResultsToClipboard = () => {
+		navigator.clipboard.writeText(JSON.stringify(results, null, 2));
+
+		openSuccessToast({
+			message: Liferay.Language.get('copied-to-clipboard'),
+		});
+	};
+
+	const _downloadResultsToTxtFile = () => {
```

Comment:
Remember to name event handlers with a `_on*` naming convention. So this could be named `_onDownloadJSON`. Same for the `_copyResultsToClipboard` (possibly `_onCopyToClipboard`). If we make these more related with what is displayed it'll make it easier to link which functions are for which buttons.

Reply:
oh right! I'll add this in!